src/app/routes.tsx

```diff
@@ -62,6 +63,12 @@ export const routes: AppRouteConfig[] = [
     path: '/plans',
     title: `${APP_TITLE} | Migration Plans`,
   },
+  {
+    component: PlanWizard,
+    exact: true,
+    path: '/planwizard',
```

Comment:
See above re: the route path

Reply:
One thing to be aware of is if we make this `/plans/create` I believe it needs to be higher in the list than `/plans/:planId` in order to prevent matching that route instead, so let's stay aware of that when this change gets rebased on/under #78.

Reply:
I've added an TODO comment too to help.